Formula/glib.rb

```diff
@@ -72,6 +73,11 @@ def install
     end
 
     bash_completion.install Dir["gio/completion/*"]
+
+    Dir[libexec/"bin/*"].each do |executable|
```

Comment:
Again, there should be a way to specify the python interpreter, in the build system, without having to wrap everything. Doing this on every formula will create a lot of headaches in the future 😢

Reply:
This is required only if formula installs python script which uses something like `#! /usr/bin/env python3`, but I can't say how many formulae like this do we have.

Then I run out of ideas for now 🙁